dev-python/cadquery-ocp/cadquery-ocp-7.5.2_rc20210706.ebuild

```diff
@@ -0,0 +1,160 @@
+# whereas this package uniquely generates one "${BUILD_DIR}/OCP/CMakeLists.txt"
+# file for each active Python version.
+inherit llvm multiprocessing python-r1
+
+MY_COMMIT="9a6210e3c490e741719ccdada900c70e58b09c91"
+OCCT_PV=$(ver_cut 1-3)
+
+DESCRIPTION="Python wrapper for OCCT generated using pywrap"
+HOMEPAGE="https://github.com/CadQuery/OCP"
+SRC_URI="https://github.com/CadQuery/OCP/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+
+# This version requirement derives from "${BUILD_DIR}/OCP/CMakeLists.txt".
+BDEPEND=">=dev-util/cmake-3.16"
+RDEPEND="~sci-libs/opencascade-${OCCT_PV}[json,tbb,vtk]"
+DEPEND="${RDEPEND}
+	>=dev-python/cadquery-pywrap-${OCCT_PV}_rc0[${PYTHON_USEDEP}]
+"
+
+S="${WORKDIR}/OCP-${MY_COMMIT}"
+BUILD_DIR="${S}"
+
+# Ensure the path returned by get_llvm_prefix() contains clang.
+llvm_check_deps() {
+	has_version -r "sys-devel/clang:${LLVM_SLOT}"
+}
+
+# OCP currently requires manual configuration, compilation, and installation
+# inspired by the conda-specific "build-bindings-job.yml" file.
+src_prepare() {
+	default
+
+	# Most recently installed version of Clang.
+	local _CLANG_VERSION="$(CPP=clang clang-fullversion)"
+
+	# Most recently installed version (excluding trailing patch) of VTK.
+	local _VTK_VERSION="$(best_version -r sci-libs/vtk)"
+	_VTK_VERSION="$(ver_cut 1-2 "${_VTK_VERSION##sci-libs/vtk}")"
+
+	# Absolute dirname of the most recently installed Clang include directory,
+	# mimicing similar logic in the "dev-python/shiboken2" ebuild. See also:
+	#     https://bugs.gentoo.org/619490
+	local _CLANG_INCLUDE_DIR="${EPREFIX}/usr/lib/clang/${_CLANG_VERSION}/include"
+
+	# Absolute filename of the most recently installed Clang shared library.
+	local _CLANG_LIB_FILE="$(get_llvm_prefix)/lib64/libclang.so"
+
+	# Absolute dirname of OCCT's include and shared library directories.
+	local _OCCT_INCLUDE_DIR="${EPREFIX}/usr/include/opencascade-${OCCT_PV}"
+	local _OCCT_LIB_DIR="${EPREFIX}/usr/lib64/opencascade-${OCCT_PV}"
+
+	# Absolute dirname of VTK's include directory,
+	local _VTK_INCLUDE_DIR="${EPREFIX}/usr/include/vtk-${_VTK_VERSION}"
+
+	# Ensure the above paths exist as a crude sanity test.
+	test -d "${_CLANG_INCLUDE_DIR}" || die "${_CLANG_INCLUDE_DIR} not found."
+	test -f "${_CLANG_LIB_FILE}"    || die "${_CLANG_LIB_FILE} not found."
+	test -d "${_OCCT_INCLUDE_DIR}"  || die "${_OCCT_INCLUDE_DIR} not found."
+	test -d "${_OCCT_LIB_DIR}"      || die "${_OCCT_LIB_DIR} not found."
+	test -d "${_VTK_INCLUDE_DIR}"   || die "${_VTK_INCLUDE_DIR} not found."
+
+	# "dev-python/clang-python" atom targeting this Clang version.
+	local _CLANG_PYTHON_ATOM="dev-python/clang-python-${_CLANG_VERSION}"
+
+	# Ensure "dev-python/clang-python" targets this Clang version.
+	has_version -r "=${_CLANG_PYTHON_ATOM}" ||
+		die "${_CLANG_PYTHON_ATOM} not installed."
+
+	# Remove all vendored subdirectories.
+	rm -rf conda opencascade pywrap || die
```

Comment:
AFAIK `-f` makes `rm` exit successfully even if the dir does not exist, so now the `|| die` is useless. I think we should just get rid of the `-f`

Reply:
Why are you always right, Andrew? Why? WHY!?!?

Embarrassingly, that was muscle memory. I always pass `-rf` together, so that's what the fingers do by themselves now. I never even knew that you *could* pass `-r` without `-f`. Of course, you can. And here we are, washed up on the lonely shore of another misbegotten code fragment.

Reply:
I only noticed this because I made the exact same mistake about a month ago :smile: . On full-auto I did `-rf` and someone complained to me that `--force` is wrong in ebuilds (which in hind-sight makes sense, it is just that the `-f` is basically glued to the `-r` in my head :P )